views/index.ejs

```diff
@@ -11,7 +11,9 @@
     <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
```

Comment:
How you print the # of items inside the cart? If cart is the name of the array, what method do we know that will tell us how many things are inside of it?